twittermap/public/javascripts/cache/services.js

```diff
@@ -0,0 +1,575 @@
+
+/*Cache module stores user requested city polygons.When Users requests these same city polygon area again we have it in cache
+and provide it to the user without sending http request.*/
+angular.module('cloudberry.cache', ['leaflet-directive', 'cloudberry.common' ])
+.service('Cache', function( $window, $http, $compile){
+
+var cachedCityPolygonTree = rbush();
+var cachedRegion ;
+var cacheSize = 0;
+var insertedTreeIDs = new Set();
+var cacheThreshold = 7500;//hard limit
```

Comment:
Did you measure the MB size for this?  I wonder whether this is a reasonable size.